adapter/src/model/enum/Format.ts

```diff
@@ -0,0 +1,34 @@
+import { CsvInterpreter } from "../../interpreter/CsvInterpreter";
+import { JsonInterpreter } from "../../interpreter/JsonInterpreter";
+import { XmlInterpreter } from "../../interpreter/XmlInterpreter";
+
+
+export class Format {
```

Comment:
No class required I think... Just
```
export const JsonInterpreter = new JsonInterpreter(); 
...
```
Import then the following way in other files:
```
import * as Format from '...'
```

Reply:
Is beeing used as parameter in FormatConfig.ts, needs to be a class i think. Unless we dont dont need it there later on